react/src/pages/ChecksPage.js

```diff
@@ -0,0 +1,1212 @@
+    const lambdas = useFetch({
+        url: Server.Url("/lambdas", environ),
+        onData: (data) => {
+            data.sort((a,b) => a.lambda_name > b.lambda_name ? 1 : (a.lambda_name < b.lambda_name ? -1 : 0));
+            return data;
+        }
+    });
+
+    useEffect(() => {
+
+/*
+        Fetch.get(Server.Url(`/checks`, environ),
+                  groupedChecks => {
+                      setGroupedChecks(groupedChecks.sort((a,b) => a.group > b.group ? 1 : (a.group < b.group ? -1 : 0)));
+                      if (groupedChecks.length > 0) {
+                          //
+                          // Choose some group as default to show.
+                          //
+                          let group = groupedChecks.find(item => item.group.toLowerCase().includes("elasticsearch"));
+                          group = group ? group : groupedChecks[0];
+                          showGroup(group);
+                      }
+                  },
+                  setLoading, setError);
+        Fetch.get(Server.Url(`/lambdas`, environ),
+                  lambdas => {
+                      setLambdas(lambdas.sort((a,b) => a.lambda_name > b.lambda_name ? 1 : (a.lambda_name < b.lambda_name ? -1 : 0)));
+                  });
+*/
+
+        refreshChecksStatus();
+
+        // This running periodically screws up the check run configuration inputs.
+        // setInterval(() =>  refreshChecksStatus(), 10000);
+
+    }, []);
+
+    const checksStatus = useFetch();
+    function refreshChecksStatus() {
+        checksStatus.refresh(Server.Url(`/checks-status`, environ));
+    }
+
+    // This is a bit tedious to unwrap. This is what a check record looks like:
+    // {
+    //     "title": "ES Disk Space Check",
+    //     "name": "elastic_search_space",
+    //     "group": "Elasticsearch checks",
+    //     "module": "system_checks",
+    //     "schedule": {
+    //         "morning_checks_1": {
+    //             "all": {
+    //                 "kwargs": {
+    //                     "dependencies": [],
+    //                     "primary": true
+    //                  },
+    //                  "dependencies": []
+    //             },
+    //             "cron": "0 6 * * ? *",
+    //             "cron_description": "06:00 AM"
+    //         }
+    //     },
+    //     "registered_kwargs": {
+    //         "time_limit": 480
+    //     }
+    // }
+    //
+    // See foursight_core/react_api.py for how this gets put together, from the
+    // original from check_setup.json and annotated with schedule info from lambdas
+    // and extra (registered) kwargs from the check_function decorators.
+    //
+    function getKwargsFromCheck(check) {
+        let kwargs = {};
+        for (let schedule_key of Object.keys(check?.schedule)) {
+            for (let schedule_env_key of Object.keys(check.schedule[schedule_key])) {
+                if (schedule_env_key.toLowerCase() === "all" || Env.Equals(schedule_env_key, environ)) {
```

Comment:
Maybe redundant with the other stuff I keep mentioning, but can we just always pass this in lowercase?